org.gahshomar.Gahshomar.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id" : "org.gahshomar.Gahshomar",
```

Comment:
The `gahshomar.org` domain does not seem to exist.

Reply:
Is this is a must? Currently the website for the app is here: https://gahshomar.github.io/gahshomar/

Reply:
Yeah, we prefer it to avoid name clashes. GitHub projects usually use `io.github` so it would be `io.github.gahshomar.gahshomar`.

Reply:
Do I need to do this rename in flathub only or in my app as well? Because the app uses `org.gahshomar.Gahshomar` in all places such as icon name, desktop file name, appdata name ...

Reply:
Yes, files need to match the app-id from manifest. @nedrichards how  strict are we on this?